osu.Game.Modes.Osu/Objects/Drawables/Pieces/SliderBall.cs

```diff
@@ -98,17 +98,18 @@ protected bool Tracking
             }
         }
 
+        private bool validTrackingTime => Time.Current >= slider.StartTime && Time.Current <= slider.EndTime;

```

Comment:
Also, let's make it `< slider.EndTime` with a strict lesser sign. This has a few nice mathematical properties and is consistent with the [inclusive, exclusive) intervals also often used with integers.